tests/test_raster.py

```diff
@@ -178,21 +183,21 @@ def test_mask_image(self):
         )
 
     def test_warp(self):
-        self.assertEqual(self.img.dataset.crs, "EPSG:4326")
+        self.assertEqual(self.img.dataset.crs, CRS.from_epsg(4326))

```

Comment:
the check goes still wrong here? It works locally, so we need to check whether we should update https://github.com/dlr-eoc/ukis-pysat/blob/master/.github/workflows/pythonapp.yml

You can read this as a reference: https://github.com/mapbox/rasterio/blob/master/docs/faq.rst#why-cant-rasterio-find-projdb-rasterio-from-pypi-versions--120

Reply:
We will skip ot for now b2a794a